internal/graphapi/apitoken_test.go

```diff
@@ -368,3 +370,38 @@ func (suite *GraphTestSuite) TestMutationDeleteAPIToken() {
 		})
 	}
 }
+
+func (suite *GraphTestSuite) TestLastUsedAPIToken() {
+
+	t := suite.T()
+
+	defer mock_fga.ClearMocks(suite.client.fga)
+
+	// setup user context
+	reqCtx, err := userContext()
+	require.NoError(t, err)
+
+	// create new API token
+	token := (&APITokenTokenBuilder{client: suite.client}).MustNew(reqCtx, t)
```

Comment:
This naming was fixed between your changes, so you need to update it to match.
```suggestion
	token := (&APITokenBuilder{client: suite.client}).MustNew(reqCtx, t)
```